app/routes/monthlychallenges/june-2022.jsx

```diff
@@ -0,0 +1,124 @@
+import { json } from '@remix-run/node';
+import { createMetaData } from '~/util/createMetaData.server';
+
+export const handle = {
+	listTitle: 'June, 2022: Giving meaningful and empathetic feedback',
+	meta: {
+		title:
+			'Monthly Theme & Challenge for June, 2022: Giving meaningful and empathetic feedback',
+		description: `June Challenge -> Being a developer is more than writing code, so this month let's work on providing each other with honest and empathetic feedback.`,
+	},
+	date: '2022-06-01',
+	hero: {
+		heroHeader: '',
+	},
+};
+
+export async function loader() {
+	const { title, description } = handle.meta;
+	return json({
+		meta: createMetaData({ title, description }),
+	});
+}
+
+export function meta({ data: { meta } = {} } = {}) {
+	return meta;
+}
+
+export default function Challenge() {
+	return (
+		<>
+			<h1>
+				<small>Monthly Theme for June, 2022:</small> Feedback
+			</h1>
+
+			<p className="lead">
+				We're challenging you to both give and accept feedback from others.
+			</p>
+			<hr />
+
+			<h2>Theme</h2>
+			<p>Giving Meaningful and Empathetic Feedback</p>
+
+			<h2>Goal</h2>
+			<p>Provide feedback four times</p>
+
+			<h3>Who can participate?</h3>
+			<p>
+				Virtual Coffee's goal is to support all developers, no matter where they
+				are in their coding journey. We encourage all members to participate.
+			</p>
+
+			<h3>Let's work together!</h3>
+			<p>
+				When we care about our community members, we can offer empathetic
+				responses that are honest and allow them to grow and to fix a problem.
+				Sometimes these conversations are hard. But sometimes we need hard
+				conversations to help us grow. Let's approach each other with kindness
+				and honesty, and allow this kind of feedback to be a regular part of our
+				process.
+			</p>
+
+			<h3>How it works</h3>
+			<p>
+				For this month's challenge we both need people asking for feedback and
+				people giving feedback! If you need feedback on content or code, please
+				post in the #monthly-challenge channel. It could be on a blog post, an
+				idea, a coding project/PR, a talk, an interview, or whatever you could
+				use feedback on.
+			</p>
+
+			<h6>Places where you can give feedback:</h6>
```

Comment:
```suggestion
			<h4>Places where you can give feedback:</h4>
```

Reply:
@redapy we definitely don't want to skip heading levels. Maybe we need some style tweaks (in a separate issue) - but for now I'd leave it as an h4 as @Andrew-Bush suggests 👍🏼

Reply:
To expand a little, we don't to skip heading levels because they have a semantic meaning and are not meant to be treated as a method of dictating style. 

This can be really important for folks who use screen readers.

Reply:
Thanks for the Infos :100: